py_driverstation.py

```diff
@@ -166,13 +172,10 @@ def connect_buttons(self):
 
     def setup_remote_ip_selector(self):
         """Setup the remote ip selector input box"""
-        self.InputIP.setText(
-            self.config.remote_ip)
+        self.InputIP.setText(self.config.remote_ip)
 
         self.UpdateIPButton.clicked.connect(
-            lambda: self.ip_input_update(
-                self.InputIP.text())
-        )
+            lambda: self.ip_input_update(self.InputIP.text()))
```

Comment:
Careful with the indents here.

Reply:
Yapf is telling me that's the right way to format the line.